Eastern-Research-Group/mywaterway
# HMW-322 Added email link to educators page.

## Related Issues:
* https://jira.epa.gov/browse/HMW-322

## Main Changes:
* Added email link to educators page.

## Steps To Test:
1. Navigate to http://localhost:3000/educators
2. Verify email link is there
